src/components/link-region/link-region.module.css

```diff
@@ -20,6 +20,7 @@
 	bottom: 0;
 	left: 0;
 	right: 0;
+	user-select: none;
```

Comment:
This fixes an issue where the `&nbsp;` in `link-region` (used to ensure links are focus-able in Safari) is select-able, created a buggy-looking appearance in certain scenarios. We know the contents of this element will always be `&nbsp;`, so we can safely disable selection.